modules/virt-deleting-templates-from-custom-namespace.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * virt/vm_templates/virt-deleting-templates-from-custom-namespace.adoc
+
+[id="virt-deleting-templates-from-custom-namespace_{context}"]
+= Deleting templates from a custom namespace
+
+To delete Red Hat templates from a custom namespace, remove the `commonTemplateNamespace` attribute and delete each template from that custom namespace.
+
+.Procedure
+
+. Open the `hco` CR to edit the file.
++
+[source,terminal]
+----
+$ oc edit hco -n openshift-cnv kubevirt-hyperconverged
+----
++
+. Remove the `commonTemplateNamespace` attribute and the custom namespace from the spec.
++
+[source,yaml]
+----
+apiVersion: hco.kubevirt.io/v1beta1
+kind: HyperConverged
+metadata:
+  name: kubevirt-hyperconverged
+spec:
+  commonTemplatesNamespace: customnamespace <1>
+----
+<1> Delete the `commonTemplatesNamespace` attribute and the value.
```

Comment:
Suggest rewording this as "The `commonTemplatesNamespace` attribute to be deleted." to avoid any confusion with the actual procedure steps.

Reply:
Updated the content.